samples/word/10-content-controls/insert-and-change-tracking-state.yaml

```diff
@@ -0,0 +1,124 @@
+order: 9
+id: word-content-controls-insert-and-change-tracking-state
+name: Get content controls by their change tracking state
+description: Inserts content controls then retrieves them by their change tracking state.
+author: xiruatms
+host: WORD
+api_set:
+    WordApi: '1.5'
+script:
+    content: |
+        $("#insert-controls").on("click", () => tryCatch(insertContentControls));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function insertContentControls() {
+          // Inserts content controls, changes them, then gets them by their change tracking state.
+          await Word.run(async (context) => {
+            let trackAddedArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.added];
+            let trackDeletedArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.deleted];
+            let trackNormalArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.normal];
+            let paragraphs = context.document.body.paragraphs;
+            paragraphs.load();
+            await context.sync();
+            paragraphs.items[0].insertContentControl("PlainText");
+            paragraphs.items[1].insertContentControl("PlainText");
+            paragraphs.items[2].insertContentControl("RichText");
+            context.document.changeTrackingMode = Word.ChangeTrackingMode.trackAll;
+            paragraphs.items[3].insertContentControl("RichText");
+            context.document.body
+              .getContentControls()
+              .getFirst()
+              .delete(false);
+            context.document.changeTrackingMode = Word.ChangeTrackingMode.off;
+            await context.sync();
+            let addedContentControls = context.document.body.getContentControls().getByChangeTrackingStates(trackAddedArray);
+            let deletedContentControls = context.document.body
+              .getContentControls()
+              .getByChangeTrackingStates(trackDeletedArray);
+            let normalContentControls = context.document.body.getContentControls().getByChangeTrackingStates(trackNormalArray);
+            addedContentControls.load();
+            deletedContentControls.load();
+            normalContentControls.load();
+            await context.sync();
+            console.log(`Added content controls: ${addedContentControls.items.length}`);
+            console.log(`Deleted content controls: ${deletedContentControls.items.length}`);
```

Comment:
These logs could read like the controls are being added and deleted. Could we make this less ambiguous?